streams/shared/src/main/scala/zio/stream/experimental/ZStream.scala

```diff
@@ -454,6 +526,21 @@ object ZStream extends Serializable {
       } yield Control(pull, Command.noop)
     }
 
+  /**
+   * The stream that never produces any value or fails with any error.
+   */
+  val never: UStream[Nothing] =
+    ZStream(ZManaged.succeedNow(Control(UIO.never, Command.noop)))
+
+  /**
+   * Constructs a stream from a range of integers (lower bound included, upper bound not included)
+   *
+   * @param min the lower bound
+   * @param max the upper bound
+   */
+  def range(min: Int, max: Int): ZStream[Any, Nothing, Any, Option[Unit], Int] =
```

Comment:
Should we set B = Unit/Any here?

Reply:
yeah, I think so, it's annoying right now.